Tests/TestingTests/TestSupport/TestingAdditions.swift

```diff
@@ -366,3 +366,27 @@ extension Trait where Self == TimeLimitTrait {
     return Self(timeLimit: timeLimit)
   }
 }
+
+extension Issue {
+  init(kind: Kind) {
```

Comment:
Also for conciseness, this could be `kind: Kind, sourceContext: SourceContext = .init()`.

I wouldn't want to reintroduce that into production code, but this is just for tests. :)

Reply:
Yes, this one _can_ be simplified as you suggest without ambiguity. Thanks